src/client/app/types/locales.ts

```diff
@@ -2,7 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-export default class Locales{
+export default class locales{
```

Comment:
Norally class names are capitalized. Is this to avoid the same name from Plotly or something else? Maybe we should refact name if this is the case.

Reply:
Reverted back to UpperCase.